fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -31,7 +31,7 @@ _supply_ uploads app metadata, screenshots, binaries, and app bundles to Google
 
 ## Setup
 
-Setup consists of setting up your Google Developers Service Account
+Setup consists of setting up your Google Developers Service Account - https://docs.fastlane.tools/actions/supply/#setup
```

Comment:
accident?

Reply:
oh I get it now. This doc is somehow imported and shown on the website. I am looking at the preview in vscode and was thinking it's broken, lol. Will undo these changes!